bower.json

```diff
@@ -2,11 +2,12 @@
   "name": "paper-ripple",
   "private": true,
   "dependencies": {
-    "core-icon": "Polymer/core-icon#master",
-    "core-icons": "Polymer/core-icons#master",
-    "font-roboto": "Polymer/font-roboto#master"
+    "core-icon": "Polymer/core-icon#0.8-preview",
+    "core-icons": "Polymer/core-icons#0.8-preview",
+    "font-roboto": "Polymer/font-roboto#master",
+    "polymer": "polymer/polymer#0.8-preview"
```

Comment:
I don't think so. If you want to use a component in your app you don't want to download all of the dependencies for the demo; you only want what it takes to run the app in production.

According to the Element Repo Org doc, `dependencies` should include only run time dependencies.